### ISSUE TITLE
`wt logs` stop working

### ISSUE THREAD

**Issue by jcenturion** [auth0/wt-cli#49]
Sometimes, I got this error `[04:02:04.231Z] ERROR wt: Unexpected end of input`. When that happends, `wt logs` stops working.

**Comment by ggoodman** [comment]
Closing because this issue only relates to `v2` and will be resolved once the branch is merged into master.

**Comment by ggoodman** [comment]
Reopening because there appear to be lingering issues.
